src/Features/Core/Portable/EditAndContinue/EditAndContinueWorkspaceService.cs

```diff
@@ -306,7 +324,7 @@ public void DiscardSolutionUpdate()
         public async ValueTask<ImmutableArray<ImmutableArray<(LinePositionSpan, ActiveStatementFlags)>>> GetBaseActiveStatementSpansAsync(Solution solution, ImmutableArray<DocumentId> documentIds, CancellationToken cancellationToken)
         {
             var editSession = _editSession;
-            if (editSession == null)
+            if (editSession == null || !editSession.InBreakState)
```

Comment:
Is editSession ever null here now? Other than when EnC isn't active at all, but these calls shouldn't happen then, right? This could be a contract check perhaps?

Reply:
I'll follow up with refactoring that cleans up edit session nullability. That can wait for Preview 3 though.